src/components/data/row/CoinRankRow.tsx

```diff
@@ -201,7 +201,7 @@ const getStyles = cacheStyles((theme: Theme) => ({
     flex: 1
   },
   negativeText: {
-    color: theme.negativeText
```

Comment:
The not right way to change the color. Negative and danger are two different themes. Don't change how they are used but rather change the color used to represent them. If there's concern that negativeText is used in too many places, then introduce a new theme such as `negativePercentText`. It's worth an audit on where negativeText is used in that it may be worth changing all uses to red anyway.

Reply:
Yeah I think we need to rethink our reddish text theme organization. I'll add a new theme key instead